docs/search/README.md

```diff
@@ -20,15 +16,13 @@ We're taking a [data informed](https://stacks.wellcomecollection.org/data-inform
 
 In broad terms, we want to produce a number which tells us whether a candidate change to the search system makes the results more or less relevant to the person searching.
 
-There's a lot of interesting discussion to be had about different search modalities, precision vs serendipity, linked open data & knowledge graphs, and existential questions about what relevance even _is_, dude, but the broad definition above is a useful, concise statement to revert back to.
+There's a lot of interesting discussion to be had about different search modalities, precision vs serendipity, linked open data & knowledge graphs, and existential questions about what relevance even _is_ but the broad definition above is a useful, concise statement to revert back to.
 
 #### Explicit vs implicit measurement
 
 We're collecting data in two distinct ways:
 
-* **Implicit data collection:** users know that their data might be used to understand aggregate user behaviour, but their behaviour on the site is driven by another motive.
-
-The larger of the two major datasets we are using is passively collected while users carry out searches and look through pages of results.
+* **Implicit data collection:** users know that their data might be used to understand aggregate user behaviour, but their behaviour on the site is driven by user search requirements and paging through results.
```

Comment:
I'm not sure I understand, a person's behaviour is driven by our research and paging? How are we driving their behaviour?

Reply:
Hmm, maybe better, "their behaviour on the site is driven by their own research requirements."